gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -158,7 +138,7 @@ public class GobblinServiceJobScheduler extends JobScheduler implements SpecCata
    * e.g. There are multi-datacenter deployment of GaaS Cluster. Intra-datacenter fail-over could be handled by
    * leadership change mechanism, while inter-datacenter fail-over would be handled by DR handling mechanism.
    */
-  private boolean isNominatedDRHandler;
+  private final boolean isNominatedDRHandler;
```

Comment:
I'm not familiar w/ multi-datacenter deployment... does that play well w/ multi-active?

Reply:
dr -> disaster recovery.
i dont think we ever used that.

Reply:
i think that functionality is also not completed. do you have more details?

Reply:
no, I don't.  I see a "skeleton" for DR was added in 2019.  for now let's leave as-is